lib/network/default.nix

```diff
@@ -0,0 +1,107 @@
+{ lib }:
+{
+  ipv6 = {
+    /**
+      Normalize an IPv6 address by removing the "::" compressions and padding
+      them with the necessary number of zeros to create the correct address. Converts
+      an address from the string to the list of strings which then can be parsed using
+      `_parseNormalized`. Throws an error when the address is malformed.
```

Comment:
Please talk about expanding and compressing, as that is the official lingo used in the RFC 4291. And make sure to reference RFC as the canonical source for these representations.

> Expand an IPv6 Address according to RFC 4291, by doing this and that